pkgs/applications/graphics/mcomix3/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchFromGitHub
+, python3
+, python3Packages
+, wrapGAppsHook
+, gobject-introspection
+, gtk3
+, gdk-pixbuf
+# Recommended Dependencies:
+, unrar
```

Comment:
If they are runtime dependencies we could wrap them. Not sure if this is nicer though.

Reply:
They are runtime.

Reply:
Something like this:

```
wrapProgram $out/bin/somafm --prefix PATH ":" "${stdenv.lib.makeBinPath [ curl jq mpv ]}";
```